client/app/components/Toast/Toast.scss

```diff
@@ -0,0 +1,22 @@
+@import "~styles/_global.scss";
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+.toast {
+  display: flex;
+  justify-content: space-between;
+  width: fit-content; 
+  margin: 0 auto;
+
+  button {
+    display: flex;
+    align-self: end;
+    background: transparent;
```

Comment:
Is this necessary?

Reply:
Added this to vertically align the cross icon but seems unnecessary so removed it.

Reply:
You'll want to remove this file!

Reply:
Let's not commit this file!